doc/api/process.md

```diff
@@ -899,6 +904,10 @@ process.env.TEST = 1;
 delete process.env.TEST;
 console.log(process.env.TEST);
 // => undefined
+process.env.TEST = 1;
+process.env.TEST = undefined;
+console.log(process.env.TEST);
```

Comment:
Instead of this, would it be better to show that the property `TEST` doesn't exist anymore in `process.env`, like in the test? As it is, it may not convey the idea clearly (setting `undefined` and then `undefined` is printed).

Reply:
like this?

```js
console.log('TEST' in process.env);
// => false
```

Reply:
Yes. That's better, right?